examples/compile-only-dependencies/coollib/BUILD

```diff
@@ -0,0 +1,13 @@
+java_library(
+    name = "coollib",
+    srcs = glob(["src/main/java/**/*.java"]),
+    deps = ["//examples/compile-only-dependencies/fancy",
+            "@maven//:com_google_guava_guava",
+    ]
+)
+
+java_binary(
+    name = "make-coollib",
+    runtime_deps = [":coollib"],
+    main_class = "com.pomgen.example.Main",
+)
```

Comment:
Maybe remove this java_binary target, since it doesn't add anything for this example?